src/Document.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+
+namespace Ipssi\Evaluation;
+
+
+class Document
+{
+    /** @var string */
+    private $name;
+
+    /** @var RGB */
+    private $color;
+
+    /** @var int */
+    private $height;
+
+    /** @var int */
+    private $width;
+
+    /** @var array */
+    private $content;
+
+    /**
+     * Document constructor.
+     * @param string $name
+     * @param RGB $color
+     */
+    public function __construct(string $name, RGB $color, int $height, int $width)
+    {
+        $this->name = $name;
+        $this->color = $color;
+        $this->height = $height;
+        $this->width = $width;
+        $this->content = [];
+    }
+
+    /**
+     * @return string
+     */
+    public function getName(): string
+    {
+        return $this->name;
+    }
+
+    /**
+     * @return RGB
+     */
+    public function getColor(): RGB
+    {
+        return $this->color;
+    }
+
+    /**
+     * @return int
+     */
+    public function getHeight(): int
+    {
+        return $this->height;
+    }
+
+    /**
+     * @return int
+     */
+    public function getWidth(): int
+    {
+        return $this->width;
+    }
+
+    /**
+     * @return array
+     */
+    public function getContent(): array
+    {
+        return $this->content;
+    }
+
+    public function toString(): string
+    {
+        $string = '======== ' . $this->getName() . ' ========' . PHP_EOL;
+        $string .= 'Color : ' . $this->getColor()->toString() . PHP_EOL;
+        $string .= 'Height : ' . $this->getHeight() . PHP_EOL;
+        $string .= 'Width : ' . $this->getWidth() . PHP_EOL;
+
+        foreach ($this->content as $item) {
+            $string .= $item->toString();
+        }
+
+        return $string;
+    }
+
+    public function addElement($element)
```

Comment:
```suggestion
    public function addElement(Content $element)
```
Ainsi, pas besoin du `if` dans la méthode